randovania/games/fusion/generator/base_patches_factory.py

```diff
@@ -37,4 +41,16 @@ def assign_static_dock_weakness(
                 )
             )
 
+        # Some areas have issues with the hatch limit of 6, so we set some to open to remove 1-way possibilities
+        docks = [
+            {"region": "Sector 2 (TRO)", "area": "Cathedral", "dock": "Door to Ripper Tower"},
+            {"region": "Sector 2 (TRO)", "area": "Ripper Tower", "dock": "Door to Cathedral"},
+            {"region": "Sector 5 (ARC)", "area": "Arctic Containment", "dock": "Door to Ripper Road"},
+            {"region": "Sector 5 (ARC)", "area": "Ripper Road", "dock": "Door to Arctic Containment"},
+        ]
+        for dock in docks:
+            dock_weakness.append(
+                (get_node(nic(dock["region"], dock["area"], dock["dock"]), DockNode), open_transition_door),
```

Comment:
Wait why is forcing everything to be open necessary if you already exclude all the doors from dlr?